editor/editor_plugin.h

```diff
@@ -87,6 +88,8 @@ class EditorInterface : public Node {
 	Array get_open_scenes() const;
 	ScriptEditor *get_script_editor();
 
+	Ref<EditorCommandPalette> get_editor_palette();
```

Comment:
Could be renamed to `get_command_palette()`. "Editor palette" might be mistaken for e.g. color palette.